src/aegis/features/cve/suggest_cwe_prompt.txt

```diff
@@ -33,6 +33,7 @@ b) Provide the standard CWE identifier (e.g., CWE-119).
 
 c) Offer a concise explanation outlining the connection between the CVE description and the predicted CWE.
 
+d) Avoid predicting CWEs that are discouraged or prohibited for Vulnerability Mapping by MITRE.  In particular, do not suggest CWE-264 and CWE-269.
```

Comment:
probably have to enhance suggest cwe feature itself as well

Reply:
it does imply that prompt management systems might have to address such model specific instructions ... though so far I have not seen anything of worth adopting (open to ideas because I think globs of text as instruction is going to get unmanageable quickly).